apps/src/sites/studio/pages/lessons/edit.js

```diff
@@ -73,6 +74,8 @@ $(document).ready(function() {
     activities.push(emptyActivity);
   }
 
+  objectives.forEach(objective => (objective.key = objective.id + ''));
```

Comment:
Why do you need this?

Reply:
Objectives don't have a unique key associated with them, so I need to give them one for the react component. It's modeled off of what's done for activities above. I'll add a comment.